services/user.py

```diff
@@ -0,0 +1,37 @@
+from db.models import User
+
+
+def create_user(username: str, password: str,
+                email: str = None, first_name: str = None,
+                last_name: str = None) -> User:
+    user = User.objects.create_user(username=username, password=password)
```

Comment:
Consider handling potential exceptions when creating a user, such as `IntegrityError` if the username already exists. This will help in providing a more user-friendly error message or handling it appropriately.